code/obj/item/storage/ammo_pouches.dm

```diff
@@ -133,6 +133,13 @@
 	/obj/item/item_box/medical_patches/mini_silver_sulf,
 	/obj/item/item_box/medical_patches/mini_synthflesh)
 
+/obj/item/storage/medical_pouch/medic //used by regular medical doctors
+	name = "medical pouch"
+	icon_state = "ammopouch-medic"
+	w_class = 1.00
+	slots = 4
+	does_not_open_in_pocket = 0
```

Comment:
Even the nerfed version seems ridiculously good - you're essentially giving doctors another 3 pockets. Strong disagree on adding this.